### PR TITLE
Split `onadmin_proposal`

### PR DESCRIPTION
**Pull request by vmoravec**
Split `onadmin_proposal` into several standalone functions to allow deploying a single proposal by calling e.g. `deploy_single_proposal keystone` after sourcing the qa_crowbarsetup. 

I extracted the code from `onadmin_proposal` that was not specific to a proposal into `prepare_proposals` that is about to be called only once before messing up with the barclamps' installations. Also added `set_dashboard_alias` and `reboot_nodes_via_ipmi` functions that were extracted from the `onadmin_proposal` and `onadmin_allocate` function.

Tested  successfully.

Note: these changes are refactoring of the existing code, no new functionality has been added.

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1248,34 +1248,41 @@ function set_node_role_and_platform()
     rm -f $t
 }
 
+# Reboot the nodes with ipmi
+function do_ipmi()
```
Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: this function name is too generlc, the name should say what the function is doing
maybe something ike `reboot_nodes_via_ipmi`

**Comment by bmwiedemann** [comment]
The indention fix should be squashed into the main commit via git rebase --interactive master

**Comment by vmoravec** [comment]
Method renamed, commits squashed.

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -1248,34 +1248,41 @@ function set_node_role_and_platform()
...
-            local pw
-            for pw in 'cr0wBar!' $extraipmipw ; do
-                local ip=$bmc_net.$(($ip4 + $i))
-                (ipmitool -H $ip -U root -P $pw lan set 1 defgw ipaddr "$bmc_net.1"
-                ipmitool -H $ip -U root -P $pw power on
-                ipmitool -H $ip -U root -P $pw power reset) &
-            done
-        done
-        wait
+        do_ipmi
```
Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: forgot to rename the call of the function :)

**Comment by jdsn** [comment]
:+1: besides the wrong call of the function

**Comment by vmoravec** [comment]
Use correct function name `eboot_nodes_via_ipmi`

**Comment by bmwiedemann** [comment]
merged

**Comment by aspiers** [comment]
Belated :+1: - great refactoring thanks!

**Pull request closed without merging**